src/app/Library/CrudPanel/Traits/Query.php

```diff
@@ -148,4 +150,105 @@ public function orderByWithPrefix($column_name, $column_direction = 'ASC')
 
         return $this->query->orderBy($column_name, $column_direction);
     }
+
+    /**
+     * return the nested query columns.
+     *
+     * @param  \Illuminate\Database\Eloquent\Builder  $query
+     * @return array
+     */
+    public function getNestedQueryColumns($query)
```

Comment:
Does this really need to be `public`?

```suggestion
    private function getNestedQueryColumns($query)
```